lma-websocket-stack/deployment/lca-websocket.yaml

```diff
@@ -164,6 +164,12 @@ Parameters:
     Type: String
     Description: LCA settings SSM Parameter store 
 
+  AssistantWakePhraseRegEx:
```

Comment:
worth adding a comment to the template to say why we added this parameter - otherwise some diligent person might reasonably conclude that since it's not actually used in the Custom Resource Lambda it can be removed from this template.

Reply:
+1